.github/workflows/helm.yaml

```
@@ -0,0 +1,46 @@
+name: Helm
+on:
+  push:
+    tags: ["*"]
+  pull_request:
+
+jobs:
+  chart:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set the version (tag)
+        if: github.event_name == 'push'
+        run: echo "VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+      
+      - name: Set the the mock version (PR)
+        if: github.event_name == 'pull_request'
+        run: echo "VERSION=PR-${GITHUB_SHA}" >> $GITHUB_ENV
+      
+      - name: Replace the version in Chart.yaml
+        run: |
+          sed -i "s/version: .*/version: $VERSION/" kubernetes/helm/Chart.yaml
+      
+      - name: Print the changes
+        run: cat kubernetes/helm/Chart.yaml
+      
+      - name: Replace image versions in values.yaml
+        run: |
+          sed -i "s/version: .*GHA/version: $VERSION/" kubernetes/helm/values.yaml
+          sed -i "s/wgVersion: .*GHA/wgVersion: $VERSION/" kubernetes/helm/values.yaml
+          sed -i "s/nginxVersion: .*GHA/nginxVersion: $VERSION/" kubernetes/helm/values.yaml
+
+      - name: Push Helm chart to registry
+        if: github.event_name == 'push' # skip on PRs
```

Why not unify the approach from other file:
```suggestion
        if: github.event_name != 'pull_request' # skip on PRs
```

or just align other file to `== "push"` - seems more intuitive if other `on:` cases appear.

https://github.com/glothriel/wormhole/blob/57-release-1-0-0/.github/workflows/docker.yaml#L78